frontend/src/views/LoginView.vue

```diff
@@ -18,20 +18,23 @@ async function login(credentials) {
     })
 
     if (response.ok) {
+      error.value  = ""
       await router.push({
         name: "home",
         query: { loginSuccess: null }
       })
     }
+    else {
```

Comment:
the else case should be on the same like as the closing if case

```suggestion
    } else {
```